src/main/resources/db/migration/common/V1_403__automated_task_users.sql

```diff
@@ -0,0 +1,94 @@
+INSERT INTO user_account
+VALUES (-6, NULL, 'System user for ProcessDailyList Job', NULL,
+        'systemProcessDailyListAutomatedTask', '2024-01-01 00:00:00+00', '2024-01-01 00:00:00+00', NULL, 0, 0, NULL, true,
```

Comment:
Dont know why, but see 31 tasks but 28 in the code base

Reply:
There were 3 new ones added to Flyway last week, so this is expected.